studio/queues/local_queue.py

```diff
@@ -23,11 +23,11 @@ def __init__(self, path=None, verbose=10):
         self.logger.setLevel(verbose)
         self.status_marker = os.path.join(self.path, 'is_active.queue')
         try:
-            with open(self.status_marker, "w") as sm:
-                pass
-        except IOError as error:
-            self.logger.error('FAILED to create {0} for LocalQueue. ABORTING.'
-                              .format(self.status_marker))
+            with open(self.status_marker, "w") as smark:
+                _ = smark
```

Comment:
Interesting lint stuff

Reply:
Just so pylint doesn't complain about unused value "smark". And "_" is a special "I don't care" identifier.